src/lazy/responses.ts

```diff
@@ -1,20 +1,12 @@
-import {
-  $Responses,
-  $$Responses,
-  is$$Responses,
-  is$Reference
-} from "../generated/lazy";
+import { $Responses, is$$Responses, is$Reference } from "../generated/lazy";
 import { Responses } from "../generated/full";
 import _response from "./response";
 
 export default (o: $Responses): Responses =>
   is$$Responses(o)
     ? Object.entries(o)
         .map(([a, b]) => ({
-          [a]:
-            is$Reference(b) && b.$ref.substring(0, 11) === "#/responses"
-              ? b
-              : _response(b)
+          [a]: is$Reference(b) ? b : _response(b)
```

Comment:
Other objects are not checked against the reference path (as specified [here](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md#componentsObject)), so might as well remove it from `Response`.

Reply:
Reintroduced (with a more specific test). Seems like this is a "stop condition" for other reference parsing.